scrapy/core/scheduler.py

```diff
@@ -1,16 +1,31 @@
-import os
+from abc import ABC
```

Comment:
I don't think it works in Python 2

Reply:
Don't know enough about `ABC`.  Perhaps we can just leave out this part and have it inherit from `object`?